src/common/_modules/application_gateway/main.tf

```
@@ -647,6 +647,11 @@ module "app_gw" {
           backend               = "platform-api-gateway",
           rewrite_rule_set_name = "rewrite-rule-set-api-app-rewrite-platform-legacy"
         },
+        api-gateway-platform-cdc = {
```

Only `cdc` API, platform in here refer to `platform` api group and not to `platform` api gateway.
```suggestion
        api-gateway-cdc = {
```